contract.php

```diff
@@ -269,3 +269,52 @@ function contract_civicrm_post($op, $objectName, $id, &$objectRef){
     }
   }
 }
+
+/**
+ * Adjust output of membership custom data for contract file download
+ * @param $content
+ * @param $context
+ * @param $tplName
+ * @param $object
+ */
+function contract_civicrm_alterContent(  &$content, $context, $tplName, &$object ) {
```

Comment:
@mattwire Could we have another look at why the jQuery approach doesn't work? ``civicrm_alterContent`` should really be a last resort - it's fragile _and_ pretty slow.